extensions/vault/deployment/pom.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-vault-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-vault-deployment</artifactId>
+    <name>Quarkus - Vault - Deployment</name>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vault</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-arc-deployment</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jackson-deployment</artifactId>
+        </dependency>
+
+        <!-- test -->
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-vault-test</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+
+    </dependencies>
+
+    <build>
+        <testResources>
+            <testResource>
+                <directory>src/test/resources</directory>
+                <filtering>true</filtering>
+            </testResource>
+        </testResources>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <annotationProcessorPaths>
+                        <path>
+                            <groupId>io.quarkus</groupId>
+                            <artifactId>quarkus-extension-processor</artifactId>
+                            <version>${project.version}</version>
+                        </path>
+                    </annotationProcessorPaths>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
Do we have IT tests here? They are usually in a separate `integration-tests` directory. Maybe it's a special case?

Reply:
these are lightweight 
@sberyozkin had the same comment twice 

I tried to explain it in https://github.com/quarkusio/quarkus/pull/2897#issuecomment-530129859:

> having AgroalVaultITCase and AgroalVaultKv2ITCase in agroal-deployment allows to test plenty of configurations that are directly related to agroal. I saw that as an advantage to keep them in the agroal project (same as VaultITCase in vault-deployment), specifically when using testcontainers, which provides ease of use even from the ide.

it just felt overkill to create separate integration tests modules for vault and agroal. if there had been a java in memory vault impl (such as h2 for db), I would have used from unit tests, and not considered the tests as being IT, just like hibernate-orm is doing with h2. 
anyway, you are the second person to feel surprised. so may be it is actually better to move them like you are suggesting. can you confirm? thanks.

Reply:
I ended up moving the IT to specific modules in integrations-tests (vault, vault-agroal and vault-app with the native test)
cc @sberyozkin